src/coverlet.console/Program.cs

```diff
@@ -17,15 +20,70 @@
 using Coverlet.Core.Helpers;
 using Coverlet.Core.Reporters;
 using Coverlet.Core.Symbols;
-using McMaster.Extensions.CommandLineUtils;
 using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+using Newtonsoft.Json;
 
 namespace Coverlet.Console
 {
-    class Program
+    public static class Program
     {
-        static int Main(string[] args)
+        public static async Task<int> Main(string[] args)
```

Comment:
async Task is not needed if we don't await nothing, I think we can keep old signature